e2e/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+var TRACE_API_KEY_TEST = 'api-key'
+var TRACE_SERVICE_NAME_TEST = 'service-name'
+
+var childProcessTest = require('./utils/childProcessTest')
+var test = require('tape')
+var waterfall = require('async').waterfall
+var path = require('path')
+
+test('Trace collector', function (t) {
+  waterfall([
+    childProcessTest.bind(
```

Comment:
tried very hard to keep this code unclutter, this is the best i managed to achieve